pkg/tenant/auth_test.go

```diff
@@ -440,3 +440,57 @@ func (s *tenantTestSuite) TestVerifyEmail() {
 	err = s.client.VerifyEmail(ctx, req)
 	s.requireError(err, http.StatusBadRequest, "invalid token")
 }
+
+func (s *tenantTestSuite) TestResendEmail() {
```

Comment:
I have to say the tests with the subtests are MUCH more readable.

Reply:
Agreed, they are a lot easier to review